### PR TITLE
Shipments table

### PR DESCRIPTION
**Pull request by atkinson-daniel**
# PR Description 
Please include a summary of the change:

Created the Shipment table with a delivery date, crate id, and user id

Commit Emojis:

If it is a new feature, describe the feature and the iteration of the project spec it addresses.

Shipment table will be used to update delivery date and show a list of products

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: code/api/src/migrations/8-shipments.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      },
+      crateId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'crates',
+          key: 'id'
+        },
+        allowNull: false
+      },
+      deliveryDate: {
```
Comment (infamouskeyduster) [inline code/api/src/migrations/8-shipments.js:28]: Alright! We have our delivery date!

File: code/api/src/migrations/8-shipments.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
```
Comment (infamouskeyduster) [inline code/api/src/migrations/8-shipments.js:36]: I wish I knew more of the back-end and what I was looking at here… Is this our mutator that is going to allow our user to update their date?

File: code/api/src/modules/crate/model.js [hunk 3]
```diff
@@ -12,7 +12,8 @@ module.exports = function(sequelize, DataTypes) {
 
   Crate.associate = function(models) {
     Crate.hasMany(models.Subscription)
+    Crate.hasMany(models.Shipment)
```
Comment (Karlfunhouse) [inline code/api/src/modules/crate/model.js:15]: Looks like this is going to allow for multiple shipments to be created

File: code/api/src/migrations/8-shipments.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+module.exports = {
```
Comment (Karlfunhouse) [inline code/api/src/migrations/8-shipments.js:3]: Looks like all the info that each shipment is going to need to have attached.

**Comment by infamouskeyduster** [comment]
Code has been reviewed (cursory glance, because I don't know enough about the backend to really understand what I am looking at) but, it looks like all columns are added that will be needed for a user to see and then update their delivery date! Good work!

**Pull request merged**